cookbook/README.md

```diff
@@ -0,0 +1,11 @@
+# Temporal Cookbook
+
+## Construction
+
+### [Absolute from legacy Date](./absoluteFromLegacyDate.mjs)
```

Comment:
It'd be nice if we could have a nice documentation page that included code samples inline, but we can build this later as a second step.

Reply:
From what i understand we can generate a documentation page from these individual files at a later point, so im going to resolve this for now